tests/tests/checks/snapshots/analysis/java/InlineObject.kt/check_glob@method__json.snap

```diff
@@ -1,41 +1,18 @@
 ---
-source: tests/test_check_insta.rs
+source: src/bin/test-index.rs
```

Comment:
Is this a new thing from some other stack?

Reply:
Oops yes this is from https://github.com/nicolas-guichard/mozsearch/commit/462cad0e061e5f889f8517bda2103487286b003f in my nixified branch.